audit-triplestore-web/pom.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.fcrepo.camel</groupId>
+    <artifactId>fcrepo-camel-toolbox</artifactId>
+    <version>4.1.2-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.fcrepo.camel.audit</groupId>
+  <artifactId>audit-triplestore-web</artifactId>
+  <packaging>war</packaging>
+
+  <name>Web-deployed Audit System with a Triplestore backend</name>
+  <description>Camel-based audit system built with a triplestore backend, deployable to web containers</description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+  </properties>
+
+  <dependencies>
+
+    <!-- Camel Dependencies -->
+    <dependency>
```

Comment:
We should likely move the common dependencies between this pom.xml and `audit-triplestore/pom.xml` up to the top-level `pom.xml/dependencyManagement`.